README.md

```diff
@@ -8,7 +8,11 @@
 There are two modules for handling SRT streaming.
 
 #### Relay
-Hwangsae Relay performs the SRT relay. Edge devices send one SRT streaming to Relay which then can be consumed by N different SRT sinks.
+Hwangsae Relay performs the SRT relay. Edge devices send one SRT streaming to Relay which then can be consumed by N different SRT sinks. It uses:
+
+* [**chamge**](https://github.com/hwangsaeul/chamge): to register to the network and receive messages
```

Comment:
"the network" is too broad a term. "to register to the message broker and receive messages" at the very least would be more concrete.

Reply:
Noted.